src/app/education/education.component.html

```diff
@@ -1,4 +1,25 @@
-<ul>
-  <li>The University of Texas at Austin</li>
-  <li>The University of Texas Health Science Center at Houston</li>
-</ul>
+<div class="education">
+
+  <div class="row">
+      <div class="col-sm-offset-2 col-sm-4">
+        <img width="200" [src]="utLogo">
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Add an "alt" attribute to this image. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://52.38.14.86:9000/sonar/coding_rules#rule_key=Web%3AImgWithoutAltCheck)